assets/js/components/TriggerChecksExecutionRequest/TriggerChecksExecutionRequest.test.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { screen, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { withState, renderWithRouter } from '@lib/test-utils';
+
+import TriggerChecksExecutionRequest from './TriggerChecksExecutionRequest';
+
+describe('TriggerChecksExecutionRequest', () => {
+  it('should dispatch execution requested on click', async () => {
+    const user = userEvent.setup();
+
+    const initialState = {
+      clustersList: {
+        clusters: [
+          {
+            id: 'cluster1',
+            selected_checks: ['check1', 'check3'],
```

Comment:
Can we use faker for these?

Reply:
Done: https://github.com/trento-project/web/pull/1056/commits/57a90f03cab3f41f78f0cdaa467cc8a5b3db4b3c